@types/window.d.ts

```diff
@@ -38,5 +39,13 @@ declare global {
 				pageViewId: string;
 			};
 		};
+		/**
+		 * Here we want to type the google object that will be added to window.
+		 * Since the imported google namespace is a value rather than a type, use typeof.
+		 */
+		google: typeof google;
```

Comment:
where does the `google` type come from?

Reply:
(used to come from https://github.com/guardian/atoms-rendering/blob/6b4a78543898acf5fde44203b2b5a6ef693daec0/src/ima.d.ts)